### DISCUSSION TITLE
logo in navbar

### DISCUSSION THREAD

**Discussion by SanderDevisscher** [inbo/reporting-rshiny-grofwildjacht#243] [Q&A]
@dbemelmans @mvarewyck , @jrhillae en ik zijn een simpele rshiny applicatie aan het maken en we vroegen ons af welke code we in de grofwildapp gebruiken om het inbo - logo weer te geven in de navbar ?

**Comment by mvarewyck** [comment]
Relevant piece of code is here: https://github.com/inbo/reporting-rshiny-grofwildjacht/blob/master/reporting-grofwild/inst/ui/ui.R#L31-L38
Logo file itself must be in inst/ui/www/logo.png
